src/checkoutservice/main.go

```diff
@@ -109,6 +132,14 @@ func initStats(exporter *stackdriver.Exporter) {
 }
 
 func initTracing() {
+	// This is a demo app with low QPS. trace.AlwaysSample() is used here
+	// to make sure traces are available for observation and analysis.
+	// In a production environment or high QPS setup please use
+	// trace.ProbabilitySampler set at the desired probability.
+	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
+
+	initJaegerTracing()
+
```

Comment:
i think we need to move rest of this method to `initStackdriverTracing()` and call it from here. what do you think?

(ditto for all go services, sorry for not caching this earlier)